test/provision/manifest/1.14/coredns_deployment.yaml

```diff
@@ -71,13 +73,19 @@ data:
             fallthrough in-addr.arpa ip6.arpa
         }
         proxy cilium.test 10.96.0.100:53 {
-          fail_timeout 10s
-          max_fails 0
+            fail_timeout 10s
+            max_fails 0
         }
         proxy . /etc/resolv.conf {
-          fail_timeout 10s
-          max_fails 0
+            fail_timeout 10s
+            max_fails 0
         }
+        prometheus :9153
+        forward . /etc/resolv.conf
```

Comment:
I'm not agree here, there is reasons why I delete this. This is already in the proxy side, and forward can't be declared two times.